DigitalLearningSolutions.Web/Controllers/LearningPortalController/SelfAssessment.cs

```diff
@@ -209,14 +210,33 @@ public IActionResult SupervisorComments(int selfAssessmentId, int competencyNumb
             return View("SelfAssessments/SupervisorComments", model);
         }
 
+        public IActionResult SearchInSelfAssessmentOverviewGroups(SearchSelfAssessmentOvervieviewViewModel model)
+        {
+            return SelfAssessmentOverview(model.SelfAssessmentId, model.Vocabulary, null, model);
+        }
+
+        public IActionResult AddSelfAssessmentOverviewFilter(SearchSelfAssessmentOvervieviewViewModel model)
+        {
+            if (model.FilterBy == "CLEAR")
+            {
+                model.AppliedFilters.Clear();
+                model.FilterBy = nameof(model.ResponseStatus);
+            }
+            else
+            {
+                string filterName = Enum.GetName(model.ResponseStatus.GetType(), model.ResponseStatus);
+                if (!model.AppliedFilters.Any(f => f.FilterValue == model.ResponseStatus.ToString()))
+                {
+                    model.AppliedFilters.Add(new AppliedFilterViewModel(model.ResponseStatus.ToString(), null, model.ResponseStatus.ToString()));

```

Comment:
Using the text from the SelfAssessmentResponseStatus enum, results in the labels in the filters applied tags containing no spaces - could we have user-friendly labels with spaces, please?